src/utils.ts

```diff
@@ -1,5 +1,21 @@
 import { useLocation } from 'react-router';
+import { parse, format } from 'date-fns';
+import { ko } from 'date-fns/locale';
 
 export function useQuery() {
   return new URLSearchParams(useLocation().search);
 }
+
+export function changeStringToDate(dateString: string) {
+  const date = parse(dateString, 'yyyy-MM-dd', new Date());
+  return date;
```

Comment:
요렇게 하면 의미없는 변수 선언을 하나 줄일 수 있을 것 같아요!

```suggestion
  return parse(dateString, 'yyyy-MM-dd', new Date());
```